vaadin-text-field-flow-parent/vaadin-text-field-flow/src/main/java/com/vaadin/flow/component/textfield/AbstractNumberFieldI18n.java

```diff
@@ -0,0 +1,63 @@
+
+/*
+ * Copyright 2000-2024 Vaadin Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.vaadin.flow.component.textfield;
+
+import java.io.Serializable;
+
+/**
+ * The internationalization properties for {@link AbstractNumberField}.
+ */
+public interface AbstractNumberFieldI18n extends Serializable {
```

Comment:
I don't think the `Abstract` prefix fits right now because it's an interface instead of an abstract class. On the other hand, I have been trying to come up with another name but wasn't able to have many ideas. Maybe `BaseNumberFieldI18n` or just `NumberFieldI18n` (which would require changing the name of its default implementation)? Or even `HasNumberFieldErrorMessages`?

Reply:
When we were talking with @sissbruecker about converting the class into an interface, we also had trouble finding a good name. We concluded that `AbstractNumberFieldI18n` is better because it's more aligned with `AbstractNumberField`, which it is mainly associated with.